hack/tools/e2e-profiling/USAGE_EXAMPLES.md

```diff
@@ -0,0 +1,315 @@
+# Usage Examples
+
+## Example 1: Analyze Existing Test Data
+
+The repository already has test data from the OpenAPI caching optimization:
+
+```bash
+# Set up symlinks to existing data
+ln -s /home/tshort/experimental-e2e-testing/test1 e2e-profiles/baseline
```

Comment:
perhaps we should make some more abstract paths instead of publishing `/home/tshort` ?